core/kernel/tee_ta_manager.c

```diff
@@ -34,6 +35,24 @@
 #include <utee_types.h>
 #include <util.h>
 
+#if defined(CFG_TA_STATS)
+#define MAX_DUMP_SESS_NUM	(16)
+struct tee_ta_dump_stats {
+	TEE_UUID uuid;
+	uint32_t panicked;	/* True if TA has panicked, written from asm */
```

Comment:
`/* True if TA has panicked */` is enough

Reply:
done